File: Totality.java
public class Totality {
    public int sum(int[] a, String stype) {
        int fin = 0;
        if (stype.equals("even")) {
            for (int i = 0; i < a.length; i++) {
                if (i % 2 == 0) {
                    fin += a[i];
                }
            }
        } else if (stype.equals("odd")) {
            for (int i = 0; i < a.length; i++) {
                if (i % 2 != 0) {
                    fin += a[i];
                }
            }

        } else {
            for (int i : a) {
                fin += i;
            }
        }
        return fin;
    }
}

File: totality.py
def totality(vals, typ):
    final = 0
    if typ == "even":
        for i in vals:
            if vals.index(i) % 2 == 0:
                final += i
    elif typ == "odd":
        for j in vals:
            if (vals.index(j)) % 2 != 0:
                final += j
    else:
        final = sum(vals)
    return final


print(totality([1, 2, 3, 4, 5], "odd"))
